_i18n/nb-no/resources/user-guides/restore_from_keys.md

```diff
@@ -0,0 +1,52 @@
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+
+Å gjenopprette en lommebok fra pivate nøkler er ganske enkelt. Hvis du har den nødvendige informasjonen, kan du med denne guiden gjenopprette lommeboken din fullstendig. Merk: du trenger IKKE passordet ditt for å gjenopprette fra nøkler.
+
+Du trenger tre stykker data fra lommeboken din eller din .keys-fil som bevarer denne informasjonen og passordet for å avkryptere det. De tre lommebokkomponentene du trenger er:
```

Comment:
Maybe use dekryptere/avkode instead of avkryptere

Reply:
Fixed